### ISSUE TITLE
[Bug]: Storybook 7 Beta - Error: Cannot parse JSDoc tags.

### ISSUE THREAD

**Issue by reintroducing** [storybookjs/storybook#20369]
### Describe the bug

I am upgrading to the 7 beta and ran across this error. I made no code changes from 6.4.x that I was using and a few of my components are erroring with this. This is an example of the offending code:
```js
/**
     * The handler to execute when the avatar element is clicked.
     *
     * @param {SyntheticEvent} evt - The React `SyntheticEvent`.
     */
    onClick: PropTypes.func,
```

As soon as I remove this jsdoc block Storybook works fine again.

**Comment by shilman** [comment]
@reintroducing can you share more about your setup? e.g. `npx sb info`? Or better yet a reproduction repo?

**Comment by reintroducing** [comment]
@shilman my apologies, I should have been more detailed.

Environment Info:

  System:
    OS: macOS 13.0.1
    CPU: (10) arm64 Apple M1 Pro
  Binaries:
    Node: 18.12.0 - ~/.nvm/versions/node/v18.12.0/bin/node
    npm: 8.19.2 - ~/.nvm/versions/node/v18.12.0/bin/npm
  Browsers:
    Chrome: 108.0.5359.124
    Firefox: 102.0.1
    Safari: 16.1
  npmPackages:
    @storybook/addon-a11y: 7.0.0-beta.13 => 7.0.0-beta.13
    @storybook/addon-docs: 7.0.0-beta.13 => 7.0.0-beta.13
    @storybook/addon-essentials: 7.0.0-beta.13 => 7.0.0-beta.13
    @storybook/addons: 7.0.0-beta.13 => 7.0.0-beta.13
    @storybook/react: 7.0.0-beta.13 => 7.0.0-beta.13
    @storybook/react-vite: 7.0.0-beta.13 => 7.0.0-beta.13
    @storybook/theming: 7.0.0-beta.13 => 7.0.0-beta.13

That being said, I tried to create a small repro and it is working fine. I suspect I need to clean up my dependencies and see why this is happening. Will try to track it down some more.

**Comment by shilman** [comment]
Yeah I'm not able to reproduce on a fresh project either. Please let me know if you figure it out!

**Comment by reintroducing** [comment]
@shilman I was able to figure it out. I am overwriting the vite config in `viteFinal` and I was adding aliases I have in my project, but what I didn't know was that there was already an alias for `assert` pre-defined there.
```
resolve: {
    preserveSymlinks: undefined,
    alias: {
      assert: '/Users/mprzybylski/Work/fe-coat/node_modules/browser-assert/lib/assert.js'
    }
  },
```

Once I realized this, i simply added on the the config.alias rather than overwriting it and now i'm good to go. Apologies for the noise.

One thing I am curious about though, and I don't want to open a separate issue if this is a known thing already, is the constant reminder of this in the console:
`Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of ResizeDetector2 which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here: https://fb.me/react-strict-mode-find-node`

In my small repro that I was using to try the stuff for this issue, I noticed it happening as well, so my assumption is you know about this and are still working on it, but wanted to confirm so I don't create any more noise for you :)

**Comment by shilman** [comment]
@reintroducing no worries about the noise. Re `findDOMNode` noise, we're on it & hopefully have a fix here https://github.com/storybookjs/storybook/pull/20368

**Comment by reintroducing** [comment]
@shilman awesome, thanks! i'll subscribe there and follow it, appreciate all the hard work that went into this release and continues to go into the development of Storybook.

**Comment by reintroducing** [comment]
@shilman sorry one last question for you, i have docs set to automatic in my main.js:
```js
docs: {
        docsPage: 'automatic',
    },
```
but there is one story in particular where i want to disable them. I've looked through the documentation for 7 beta and can't find a way to do this (or everything I've tried doesn't work). Is there a way to disable the docs on a particular story if the overall docs are set to `automatic`? I've tried many permutations of the below but to no avail.
```js
export default {
    title: 'Icons/Line',
    component: Icons,
    parameters: {
        docs: {
            docsPage: false,
        },
        options: {
            showPanel: false,
        },
    },
};
```
